### DISCUSSION TITLE
Could somebody validate I'm writing this WebQuery command correctly

### DISCUSSION THREAD

**Discussion by Gusfmm** [arendst/Tasmota#17050] [Q&A]
Could somebody lend a hand to validate this is written correctly. I'm trying to have a tasmotized Sonoff NSPanel issue a HTTP request:

http://192.168.0.37/cm?cmnd=WebQuery http://192.168.0.20:6666 POST {"deviceid":"10014a3350","data":{"switch":"on"}}

192.168.0.20:6666 is a listener server just to make sure the NSPanel (192.168.0.37) is properly issuing the POST request. The response I get is:

192.168.0.37 - - [12/Nov/2022 19:52:31] code 400, message Bad HTTP/0.9 request type ('POST')
192.168.0.37 - - [12/Nov/2022 19:52:31] "POST  HTTP/1.1" 400 -

If I send a simple GET request: 
http://192.168.0.37/cm?cmnd=WebQuery http://192.168.0.20:6666 GET
everything seems to work fine:

←[93mSat Nov 12 19:54:25 2022←[0m  GET command received from: ('192.168.0.37', 61318)
Endpoint:  HTTP/1.1
- - - - - - - - - - - - - - - - -
Headers:
 Host: 192.168.0.20:6666
User-Agent: ESP32HTTPClient
Connection: keep-alive
Accept-Encoding: identity;q=1,chunked;q=0.1,*;q=0

- - - - - - - - - - - - - - - - -
192.168.0.37 - - [12/Nov/2022 19:54:25] "GET  HTTP/1.1" 200 -

Many thanks!

**Comment by barbudor** [comment]
First make the command work as expected in the console of the issuer tasmota

For a web command, space in the command must be escaped with %20. Other characters such as `:` and `/` must also be escaped with the appropriate % ascii code

Why do you want this tasmota to issue the WebQuery from a web command. It doesn't make sense.

**Reply by Gusfmm** [reply]
Thanks much, your suggestion to try to make it work from the console first helped! 

Re- making sense..... Smartthings Hubs and their Edge drivers are conditioned to HTTP requests to LAN IPs within the same LAN the Hub is registered to. I have two interconnected routers in my property, and need to access IP devices working within the other LAN. So thought this convoluted bandage of a solution would do the trick, at least momentarily.... and it did. Got everything working with the Tasmota as the silly request reissuer.

Thanks much.
